src/content/reference/react/use-client.md

```diff
@@ -11,7 +11,7 @@ canary: true
 
 <Intro>
 
-`'use client'` marks source files whose components execute on the client.
+`'use client'` marks which components render on the client.
```

Comment:
(I realize this is a draft so not reviewing in detail but wanted to note one high-level thing…)

thought: is there a way we can be clear up top that this marks the _boundary_ between server and client components? ideally people should realize that you need serializable props and that you don't need to mark every file deep down. it's great you have FancyText but I think it could be a bit clearer throughout

(obviously my intro text you replaced did not even try to clarify this! oops. what you wrote here is a clear improvement)

Reply:
Updated to try and highlight the concept of boundary more